test/Api.Test/Api.Test.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
+    <PackageReference Include="NSubstitute" Version="4.2.1" />
+    <PackageReference Include="xunit" Version="2.4.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
+    <PackageReference Include="coverlet.collector" Version="1.2.0" />

```

Comment:
Let's exclude this code coverage package for now as we'll need to decide how we want to handle code coverage and will need/want to update our other test project(s) to use the same or similar package as well as update any client tests regarding code coverage output formats to simplify our CI/CD pipeline.

Reply:
Removed coverlet.collector; this was added automatically via the `dotnet new xunit` command, I believe, so worth looking out for in the future if we added additional test projects.